### ISSUE TITLE
Countries - Legal Standards - Screen Missing Legal Standards Heading

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#83]
The "Legal Standards" screen is missing the "Legal Standards" heading text at the top of the screen.

Examples of this issue can be found in the following countries:
- Argentina
- Brazil
- India
- Haiti
- Jamaica
- Uganda

**Comment by carolynhuang** [comment]
Please clarify which device this is happening. Headers appear on Android 7.0.

**Comment by jfc3-dol** [comment]
@carolynhuang,

The issues is with the iOS version which this set of code is for and the only one I have tested for Sweat and Toil so far. Maybe it has been updated for Android but not iOS/Apple yet.

**Comment by sanganalakshay** [comment]
This issue was in the iOS version of the app and was fixed in the code uploaded on 31st Aug

**Comment by jfc3-dol** [comment]
@sanganalakshay,

Okay, I have not gotten the new version on my device. Partly, because I have been looking at the website and other accessibility issues I need to complete too. I plan to check it tomorrow (Wednesday).

**Comment by carolynhuang** [comment]
Please let us know whether this issue still stands, as @sanganalakshay said it was addressed.

**Comment by jfc3-dol** [comment]
Looks good now.

Closing issue.
